src/NeonView.js

```diff
@@ -50,6 +50,8 @@ class NeonView {
 
       this.editor = new params.Edit(this);
     }
+
+    this.textView = new params.TextView(this);
```

Comment:
Setting up a TextView should be optional. Check if the TextView constructor is defined and initialize if it is.